yt_dlp/extractor/nrk.py

```diff
@@ -783,8 +901,7 @@ def _extract_description(self, webpage):
     def _real_extract(self, url):
         playlist_id = self._match_id(url)
 
-        webpage = self._download_webpage(url, playlist_id)
-
+        webpage = self._download_webpage(f'https://www.nrk.no/serum/api/render/{playlist_id.split("-")[-1]}', playlist_id)
```

Comment:
This change was made because `NRKPlaylistIE` essentially just looks for `.rich[data-video-id]` elements anywhere on the page. However, the related articles section below the article can sometimes contain video player elements too, making it download videos that are not part of the "playlist". Using the render html endpoint instead does not cause this issue.

Reply:
Okay makes sense. Could you add a note explaining that?